rim-page.html

```diff
@@ -0,0 +1,327 @@
+                position: absolute;
+                top: 50%;
+                bottom: 50%;                
+            }           
+            .container-2 {            
+                width: 25%;
+            }
+            .container-3 {
+                position: relative;
+                text-align: left;
+                color: white;
+            }
+            .bb-footer {
+                float: left;
+                width: 20%;
+            }
+            .bb-footer li {
+                display: block;
+            }
+            .copyright {
+                display: block;
+                width: 100%;
+                float: right;
+            }    
+        </style>
+        <script>
+            console.log("Hello world!")
+        </script>
+    </head>
+    <body>
+        <div class="container">
+            <div class="row">
+                <div class="column">
+                    <nav>
+                        <ul>
+                            <li><a href="#Logo">Logo</a></li>
+                            <li><a href="#Products">Products</a></li>
+                            <li><a href="#Solutions">Solutions</a></li>
+                            <li><a href="#Industries">Industries</a></li>
+                            <li><a href="#Partners">Partners</a></li>
+                            <li><a href="#Support">Support</a></li>
+                            <li><a href="#Blog">Blog</a></li>
+                            <li><a href="#Search">Search icon</a></li>
+                            <li><a href="#Headset">Headset icon</a></li>
+                        </ul>
+                    </nav>
+                </div>
+        </div>                                                    
+        <div class="container hero-image">
+            <div class="row">
+                <div class="column">
+                    <div class="hero-text">
+                        <h2>Everyone is working at home. Including cyberattackers.</h2>
+                        <p>The recent SolarWinds security attack proves that remote workers and unsecured endpoints leave businesses exposed. Protection is not enough. Our AI-driven security doesn’t react to cyberthreats—it prevents them. If you believe you’ve had a breach, our BlackBerry® Security Services Incident Response experts can help.</p>
+                        <button type="button">CONTACT SALES</button>
+                    </div>
+                </div>
+            </div>               
+        </div>
+        <div class="row">
+            <div class="column">
+                <main>
+                    <h2>Market Leading Platform for Enabling IoT</h2>
+                    <p>BlackBerry provides organizations with software to secure the IoT devices they use, the products they make, the data they share and their communications.</p>
+                </main>
+            </div>
+        </div>                            
+        <div class="container">
```

Comment:
Is this why we have one container "As far as using multiple containers goes, it depends on the situation. Bootstrap uses containers in some of its other components such as navbars. But generally, you should just need one container for the majority of the page as you can have multiple rows in the same container. The main reason for having a new one might be to switch between a fluid container and fixed-width container." I have looked at other websites to see if this use container-row- column. In several of the sites I do not see (inspect element) them use row or column.

Reply:
You only ever really need one container, and if you choose to use more, it's because you really understand why you are doing it.  In the code below, there is no special context to justify the use of container class over and over.  It appears that you simply copied and pasted the style you used in opentext in here, which was not exactly the instructions.  The instructions was to use index.html and replicate that structure here, which only contains one top level container class.